packages/apps/graph/generated-schema.graphql

```diff
@@ -168,21 +191,50 @@ type PageInfo {
 scalar PositiveFloat
 
 type Query {
-  account(accountName: String!, moduleName: String!): ModuleAccount!
+  """Find an account by its name and its module, such as coin."""
```

Comment:
```suggestion
  """Retrieve an account by its name and fungible, such as coin."""
```

Here we also might want to change `moduleName` to `fungible`. Same for `ModuleAccount` -> `FungibleAccount`